lib/index.js

```diff
@@ -3,7 +3,7 @@ var clone = require('clone');
 var debug = require('debug')('analytics-node');
 var noop = function(){};
 var request = require('superagent');
-require('superagent-proxy')(request);
+// require('superagent-proxy')(request);
```

Comment:
Here's the part where commenting out this line means that the browserify bundled doesn't error out on `fs.readFileSync` not existing.